Example/src/AnimatedScrollExample.tsx

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { FC } from 'react';
```

Comment:
I guess it doesn't matter much since we have `allowSyntheticDefaultImports` enabled but we can definitely transition to using a correct version of the import.

Reply:
It's a bit convoluted, as it requires using React.FC for example or doing another import statement (Eslint doesn't like that). I'd leave that as it is for the time being.